.pre-commit-config.yaml

```
@@ -42,3 +42,9 @@ repos:
     rev: 2025.01.22
     hooks:
         - id: sp-repo-review
+
+  - repo: https://github.com/woodruffw/zizmor-pre-commit
+    rev: v1.5.2
+    hooks:
+      - id: zizmor
+        args: [--pedantic,--min-severity=low]
```

```suggestion
  - repo: https://github.com/woodruffw/zizmor-pre-commit
    rev: v1.5.2
    hooks:
      - id: zizmor
        args: [--pedantic]
```

Let's stick with `--pedantic` for now (from my comment above). It's a good thing we've fixed all the low-severity issues, and adding this option would make them come up silently if new workflows are added.